test/y2network/network_manager/config_writer_test.rb

```diff
@@ -71,7 +71,7 @@
     end
 
     it "writes connections configuration" do
-      expect(conn_config_writer).to receive(:write).with(eth0_conn, nil, routes: [])
+      expect(conn_config_writer).to receive(:write).with(eth0_conn, nil, routes: [], parent: nil)
```

Comment:
I'm a bit lost here. Why do you still using keyword arguments in the test? Could you please re-check it?

Reply:
Well in fact it pass { routes: [], parent: nil } but rubocop will complain if you use brackets ;)

Reply:
Ah! Sorry! _Quite confusing_ in this case, but if Rubocop is happy then it LGTM too :)

Thanks Knut!